INSTALL.md

```diff
@@ -1,120 +1,139 @@
-* There's currently no benefit to building a 64-bit version of Gambatte-Speedrun on Windows, so 32-bit is used for the release binaries
+**NOTE:** The instructions below assume you're installing 64-bit MSYS2 and using the 32-bit MinGW toolchain (they work with other configurations, but the commands won't be exactly the same).
+
+- Choice of 32-bit vs 64-bit for MSYS2 doesn't really matter; either is fine
+- There's currently no benefit to building a 64-bit version of libgambatte on Windows, so 32-bit is used for the release binaries
 
-***\*NOTE:*** At the end of these steps, to do any build tasks related to Gambatte-Speedrun, open the MSYS2 MinGW 32-bit shell (under the MSYS2 folder in the Start menu; ***\*must be\**** this specific shell in order for building to work).
+**NOTE:** At the end of these steps, to do any build tasks related to libgambatte, open the MSYS2 MinGW 32-bit shell (under the MSYS2 folder in the Start menu; **\*\*must be\*\*** this specific shell in order for building to work).
 
 ### Basic steps
 
-*Do the following:*
+Do the following:
 
-\- Install [MSYS2](https://www.msys2.org/) by selecting the one-click installer exe for x86_64
+- Install [MSYS2](https://www.msys2.org/) by selecting the one-click installer exe for x86_64
 
-\- Run MSYS2 shell and update MSYS2 core components and packages *(copied from [here](https://www.msys2.org/wiki/MSYS2-installation/#iii-updating-packages))*:
+- Run MSYS2 shell and update MSYS2 core components and packages _(copied from [here](https://www.msys2.org/wiki/MSYS2-installation/#iii-updating-packages))_:
+
+```
+pacman -Syuu
 ```
-$ pacman -Syuu
 
 Follow the instructions. Repeat this step until it says there are no packages to update.
 See the above link if you have an older installation of MSYS2 and/or pacman.
-```
-\- Install MSYS2 packages for general build development environment:
-```
-$ pacman -S base-devel git mingw-w64-i686-zlib mingw-w64-i686-toolchain
-```
 
-### Qt-specific steps
+- Install MSYS2 packages for general build development environment:
 
-\- Acquire and install the qt5-static library environment:
-```
-$ pacman -S mingw-w64-i686-qt5-static
 ```
-\- Modify `.bash_profile` to add qt5-static binaries to `PATH`:
-```
-$ echo 'if [[ "${MSYSTEM}" == "MINGW"* ]]; then PATH="/${MSYSTEM}/qt5-static/bin:${PATH}"; fi' >> ~/.bash_profile
+pacman -S base-devel \
+          git \
+          mingw-w64-i686-toolchain \
+          mingw-w64-i686-cmake \
+          mingw-w64-i686-ninja \
+          mingw-w64-i686-zlib
 ```
 
 ### Testrunner-specific steps
 
-\- Install the `libpng` package:
+- Install the `libpng` and `python3` packages:
+
 ```
-$ pacman -S mingw-w64-i686-libpng
+pacman -S mingw-w64-i686-libpng mingw-w64-i686-python3
 ```
 
+### Visual Studio users
+
+As there is no standard package manager for Visual Studio, you will need to either manually build zlib (and libpng for test), or use a package manager such as [vcpkg](https://github.com/microsoft/vcpkg).

```

Comment:
For this, we may want to have a folder containing submodules to the libraries (and maybe add in cmake scripts for them, if they don't already have them). That way they can always be statically included (we just include the cmake script for them).

Reply:
I am not a fan of vendoring for a library, since you cannot embed a static library in another static library, we would need to install them too. Moreover, since Python is technically a dependency to run tests, it's rather impractical to vendor that.

Thankfully, it seems that starting Visual Studio 17.6, installing vcpkg is just an option like CMake. It only supports manifest mode (hence the addition of a `vcpkg.json`) but makes it much more trivial for Visual Studio users to get it setup.